order-management-service/api/src/main/java/com/microservices/ordermanagement/api/dto/PaymentDetailsDto.java

```diff
@@ -0,0 +1,22 @@
+package com.microservices.ordermanagement.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.Getter;
+import lombok.ToString;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Getter
+@ToString
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
```

Comment:
why do we need this?

Reply:
in order not to use setters

Reply:
ooh
looks like your immutability dream is finally coming true))